source/_integrations/brother.markdown

```diff
@@ -32,7 +32,8 @@ Some very old Brother printers use different data format and these models are no
 
 ## Configuring the printer
 
-To enable SNMP, navigate to the printer's web interface (for example: `http://192.168.5.6`) and turn it on under Network / Protocol / SNMP.
+To enable SNMP, navigate to the printer's web interface (for example: `http://192.168.5.6`) and turn it on under Network / Protocol / SNMP. To see this option, you'll need to log in using the password field at the top of the page. For some printers, the default password is printed on a sticker on the back of the printer, preceded by "Pwd:". If the printer does not have a password on the sticker, the default password is "initpass".
```

Comment:
```
To see this option, you'll need to log in using the password field at the top of the page.
...the default password is "initpass"
```
This is not true for all printer models. Some models do not have a password for the www panel by default.